blog/2024-02-08-year-of-agama.mdx

```diff
@@ -0,0 +1,106 @@
+---
+date: 2024-02-08 12:00:00 +00:00
+title: The Year of Agama
+description: Take a look to the Agama roadmap for 2024
+tags:
+- project
+- systems-management
+- YaST
```

Comment:
Is `YaST` tag really needed. Nothing against YaST, btw.

Reply:
No idea, it was already in the original post: https://github.com/yast/yast.github.io/blob/65a5e6a16333c8c814002f82e9798d1658fa6077/_posts/2024-02-08-year-of-agama.md?plain=1#L11